projects/cds/src/cds-models/cds-strategy-request.model.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * A model class that is used when sending a request to the Context-Driven Services Merchandising Strategy service for returning
+ * products for a carousel. There are a number of things that can affect which products are returned, based on the users current
+ * context
+ */
+export interface StrategyRequest {
+  productId?: string;
+  category?: string;
+  facets?: string;
+  date?: string;
```

Comment:
You could use the JS `Date` object here, see `projects/core/src/model/consent.model.ts` as an example

Reply:
done